sdks/jreleaser-java-sdk-commons/src/main/java/org/jreleaser/sdk/commons/ClientUtils.java

```diff
@@ -107,6 +109,8 @@ public static Feign.Builder builder(JReleaserLogger logger,
         }
 
         return builder
+            .logger(new FeignLogger(logger))
+            .logLevel(Logger.Level.FULL)
```

Comment:
First time using Feign directly, but this doesn't seem ideal, I'm not sure if this property should map back to the JReleaser log level (trace -> full, debug -> basic, etc). But that info isn't available in the JReleaserLogger interface, so I just hard-coded it to `FULL` 🤷